jsx/Form.js

```diff
@@ -1304,6 +1304,8 @@ class ButtonElement extends React.Component {
       <div className="row form-group">
         <div className={this.props.columnSize}>
           <button
+            {...(this.props.id ? {id: this.props.id} : {})}
+            {...(this.props.name ? {name: this.props.name} : {})}
```

Comment:
@HenriRabalais this.props.id just contains what the developer passes to the component for the id value. It will not supply the html button element with the complete "id: 'ex_button'" in that case. The spread operator is used because I'm returning an object: {id: 'ex_button'} and I want both the 'key' & 'value' to be inserted into the html button. When there is nothing I don't want the id: '' to be placed in it.